src/clj/game/cards/events.clj

```diff
@@ -1544,6 +1544,30 @@
                  (doseq [c (-> @state :runner :temp-nvram)]
                    (host state side (get-in @state [:runner :identity]) c {:facedown true}))))}
 
+   "Reboot"
+   (letfn [(install-cards [state side eid card to-install]
+             (if (empty? to-install)
+               (do
+                 (move state side (find-latest state card) :rfg)
+                 (effect-completed state side eid))
+               (let [f (first to-install)]
```

Comment:
You might be able to make this an `if-let` and therefore save a layer of indentation.